apps/chat-e2e/src/tests/selectPromptPanelEntity.test.ts

```diff
@@ -1435,7 +1440,7 @@ dialTest(
         await prompts.openEntityDropdownMenu(singlePrompt.name);
         await promptDropdownMenu.selectMenuOption(MenuOptions.select);
         await promptFilter.openFilterDropdownMenu();
-        await promptFilterDropdownMenu.selectMenuOption(
+        await promptFilterDropdownMenu.selectOption(
```

Comment:
why renaming if we literally use Filter**Menu**Options inside?

Reply:
thanks, fixed